src/uct/api/uct.h

```diff
@@ -1067,6 +1068,38 @@ ucs_status_t uct_md_mem_alloc(uct_md_h md, size_t *length_p, void **address_p,
  */
 ucs_status_t uct_md_mem_free(uct_md_h md, uct_mem_h memh);
 
+/**
+ * @ingroup UCT_MD
+ * @brief list of UCT memory use advices
+ *
+ */
+typedef enum uct_mem_advices {
```

Comment:
Please make the same change from "advices" to "advice" as was done for ucp_mem_advice